### PR TITLE
frontend/feat/rentals: #4998 HomeScreen Rental Banner, NightFare static data and RentalScreen Changes

### PR DESCRIPTION
**Pull request by slow-codex**
## Type of Change

- [x] Bugfix
- [x] New feature
- [x] Enhancement
- [ ] Refactoring
- [ ] Dependency updates

## Description

- Added the new homescreen banner
- Added the explainer in the rental package selection screen
- RentalScreen RateCard changes

### Additional Changes

- [ ] This PR modifies the database schema (database migration added)
- [ ] This PR modifies dhall configs/environment variables
- [ ] This PR contains API breaking changes

## Checklist

- [ ] I formatted the code and addressed linter errors `./dev/format-all-files.sh`
- [ ] I reviewed submitted code
- [ ] I added unit tests for my changes where possible
- [ ] I added a [CHANGELOG](/CHANGELOG.md) entry if applicable
- [ ] No leak detected in leakcanary

### CODE REVIEW

File: Frontend/ui-common/src/Components/Banner/Controller.purs [hunk 1]
```diff
@@ -49,7 +50,11 @@ type Config = {
   titleTextVisibility :: Boolean,
   imagePadding :: Padding,
   cornerRadius :: Number,
-  imageMargin :: Margin
+  imageMargin :: Margin,
+  actionTextBackgroundColor :: Maybe String,
```
Comment (aryan-13) [inline Frontend/ui-common/src/Components/Banner/Controller.purs:54]: Can we put image and actionText related items in their corresponding combined configs

File: Frontend/android-native/app/src/user/jatriSaathi/res/drawable/ny_ic_rental_banner.png
Comment (aryan-13) [inline Frontend/android-native/app/src/user/jatriSaathi/res/drawable/ny_ic_rental_banner.png:1]: Can we compress this image

File: Frontend/ui-common/src/Components/Banner/View.purs [hunk 2]
```diff
@@ -86,7 +91,7 @@ view push config =
             [ height WRAP_CONTENT
             , width WRAP_CONTENT
             , gravity LEFT
-            , textFromHtml "&rarr;"
+            , textFromHtml $ if isJust config.actionTextBackgroundColor then "&darr;" else "&rarr;"
```
Comment (MercyQueen) [inline Frontend/ui-common/src/Components/Banner/View.purs:94]: make this also configurable.

File: Frontend/ui-customer/src/Screens/RentalBookingFlow/RentalScreen/View.purs [hunk 3]
```diff
@@ -139,6 +139,15 @@ rentalPackageSelectionView push state =
         ]
       , IncrementDecrement.view (push <<< DistanceIncrementDecrementAC) $ incrementDecrementConfig state
       ]
+    , textView $
+      [ stroke $ "1," <> Color.grey900
+      , textFromHtml $ "<span>Every hour gets you <b>10 kms</b> in the package.<br>You can plan additional KMs upto a limit and save on extra costs if you end up exceeding the base distance.<br>You can still review your fare on the next screen before confirming the booking.<span>"
```
Comment (aryan-13) [inline Frontend/ui-customer/src/Screens/RentalBookingFlow/RentalScreen/View.purs:144]: Add translation for this text

**Review by aryan-13 — CHANGES_REQUESTED** [review]

**Review by MercyQueen — APPROVED** [review]

**Pull request merged**